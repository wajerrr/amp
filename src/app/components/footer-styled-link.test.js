import React from 'react';
import renderer from 'react-test-renderer';
import StyledFooterLink from './footer-styled-link';

describe('FooterStyledLink', () => {
  it('should match the snapshot at the first index', () => {
    const footer = renderer
      .create(
        <StyledFooterLink
          headline="testHeadline"
          href="www.ampcanonical.com"
          sectionLevel={0}
        >
          TestLink
        </StyledFooterLink>
      )
      .toJSON();
    expect(footer).toMatchSnapshot();
  });
  it('should match the snapshot at any other index', () => {
    const footer = renderer
      .create(
        <StyledFooterLink
          headline="testHeadline"
          href="www.ampcanonical.com"
          sectionLevel={1}
        >
          TestLink
        </StyledFooterLink>
      )
      .toJSON();
    expect(footer).toMatchSnapshot();
  });
});
