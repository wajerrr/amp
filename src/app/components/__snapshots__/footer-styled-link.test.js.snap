// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterStyledLink should match the snapshot at any other index 1`] = `
<a
  className="css-1gppj2d css-1cdc8m6"
  href="www.ampcanonical.com"
>
  TestLink
</a>
`;

exports[`FooterStyledLink should match the snapshot at the first index 1`] = `
<a
  className="css-1gppj2d css-7uy4do"
  href="www.ampcanonical.com"
>
  TestLink
</a>
`;
